src/components/sidebar.tsx

```diff
@@ -16,7 +18,10 @@ export default function () {
             username="Willow.eth"
             address="0x8343...3u432u32"
         >
-            <DriveContainer />
+            <DriveContainer
+                disableHoverStyles={disableHoverStyles}
```

Comment:
Ah my bad. I understood that the hover styles were for the sidebar. In this case then it can just be internal state of the DriveContainer

Reply:
this is ok, when we disable hover styles, we have to disable it for all the Drives